src/themes/teams/components/Icon/svg/icons/call.tsx

```diff
@@ -21,11 +23,5 @@ export default {
       </g>
     </svg>
   ),
-  styles: {
-    svg: ({ variables }) => {
-      return {
-        fill: variables.color,
-      }
-    },
-  },
+  styles: {},
```

Comment:
if there are no icon-specific styles (the ones that could be applied to `call` icon exclusively), this prop could be just omitted